raster-foundry/raster-vision-plugin
# Add RfLayerRasterSource

Overview
-----

This PR adds a raster source that's capable of communicating with the Raster Foundry backend to
obtain imagery.

Testing
-----

- make sure the raster source passes a sniff test from someone on the RV team
- run the notebook in `notebooks/RfLayerRasterSource.ipynb` -- to do this:
  - `AWS_PROFILE=raster-foundry docker/run --jupyter --aws` from the repository root
  - replace `YOUR REFRESH TOKEN HERE` with an actual refresh token for RF staging
  - mash `shift + enter` until you get to the bottom

Closes #2
